Assets/MixedRealityToolkit.SDK/Experimental/HandCoach/Scripts/MoveToTarget.cs

```diff
@@ -0,0 +1,197 @@
+﻿using System.Collections;
+using System.Net.NetworkInformation;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Experimental.UI
+{
+    /// <summary>
+    /// This class provides functionality to move the hand hint from a tracking position to a target position over time.
```

Comment:
This class sounds very similar to a solver...it would be nice if it could somehow use the solver so that we do not have duplicated classes if we put this into non-experimental. Imagine you're a new developer and you see this class, and also solver. You might wonder "which one should I use?". I imagine this might be very specific to the needs of Hand Coach. In that case, could you perhaps put the class in its own namespace (`Microsoft.MixedReality.Toolkit.Experimental.UI.HandCoach`)?

Reply:
I'll use the namespace you suggested!